raster/rt_core/rt_statistics.c

```diff
@@ -1032,7 +1032,7 @@ rt_band_get_quantiles_stream(
 	uint32_t j = 0;
 	uint32_t k = 0;
 	uint32_t x = 0;
-	uint32_t y = 0;
+	int64_t y = 0;
```

Comment:
why is x different from y?

Reply:
x goes from 0 to `band->width` but y is initially setup as  `-1` and then it can either be replaced by an uint32_t value or added to a random bounded uint32_t. 

To be honest, it is a 500 LOC function and reviewing how and this could be changed to use unsigned would take a lot of time. I'm good with doubling the precision and keeping the sanitizer happy.